docs-ref-conceptual/azure-cli-variables.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Specifying values in Azure CLI commands
+description: Learn about the ways you can pass values to Azure CLI commands, including variables and ways to reuse common values.
+author: dbradish-microsoft
+ms.author: dbradish
+ms.service: azure-cli
+ms.topic: how-to
+ms.date: 03/01/2021
+ms.custom: template-how-to
+---
+
+# Specifying values in Azure CLI commands
+
+In addition to specifying values directly in a command, you can provide values in several ways:
+
+* Use shell variables
+* Set a subscription for use in multiple commands
+* Create default values for some parameters
+* Use persistent values for some parameters
+
+This article discusses various ways to specify values in Azure CLI commands.
+
+[!INCLUDE [azure-cli-prepare-your-environment.md](includes/azure-cli-prepare-your-environment.md)]
+
+## Use shell variables
+
+Azure CLI runs in a shell. This article uses Bash. For information about other shells, see [Use Azure CLI effectively](/cli/azure/use-cli-effectively). You can use variables in Bash to pass values for parameters to commands. Using variables also allows reuse of commands, either piecemeal or in scripts.
+
+This example assigns values to several variables, using standard Bash syntax:
+
+```azurecli
+MyResourceGroup=ContosoRGforVM
+MySubscription="Contoso subscription"
+vmName=VM01
+```
+
+This example uses those variables in a command to get another value and assign it to the **osType** variable:
+
+```azurecli
+osType=$(az vm get-instance-view --resource-group $MyResourceGroup \
+   --name $vmName --subscription $MySubscription \
+   --query 'storageProfile.osDisk.osType' --output tsv)
+```
+
+This command uses the `tsv` output format, which returns values without extra formatting, keys, or other symbols. Some output formats include structure or characters like quotation marks. For more information, see [Output formats for Azure CLI commands](/cli/azure/format-output-azure-cli).
+
+You can use the variable in a later command:
+
+```azurecli
+az disk create --resource-group $MyResourceGroup --name DestinationDisk \
+   --source "/subscriptions/$MySubscription/resourceGroups/$MyResourceGroup/providers/Microsoft.Compute/snapshots/MigrationSnapshot" \
+   --os-type $osType
+```
+
+## Set a subscription
```

Comment:
@TimShererWithAquent , I'll need to come back to these additional H2s for later review.